.github/workflows/builds.yaml

```diff
@@ -231,12 +231,21 @@ jobs:
         run: |
           echo "CFLAGS=-Wno-deprecated ${CFLAGS}" >> "$GITHUB_ENV"
 
+        # We use the `unstable-external_project` Meson module to enable certain
```

Comment:
I vehemently disagree with the Meson direction here. Tried to plead my case. No dice. Oh well, now we get this hack!

Reply:
https://github.com/mesonbuild/meson/pull/11053

Reply:
Merged with just the change to fatal-warnings, as originally discussed when you pled your case. :) This should do everything you need it to do.

Reply:
Right I will change this when 0.64.1 releases